app/src/main/java/org/kiwix/kiwixmobile/zim_manager/ZimManageViewModel.kt

```diff
@@ -116,8 +116,6 @@ class ZimManageViewModel @Inject constructor(
   val requestFiltering = BehaviorProcessor.createDefault("")
   val currentPage = PublishProcessor.create<Int>()
 
-  val libraryTabIsVisible = currentPage.map { it == 1 }.filter { it }
```

Comment:
Keep pulling the thread, currentPage is no longer used I believe

Reply:
Correct!